aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -1149,13 +1149,13 @@ module aptos_framework::account {
         let addr: address = @0x1234; // Define test address
         create_account(addr); // Initialize account resource
         // Assert sequence number intializes to 0
-        assert!(borrow_global<Account>(addr).sequence_number == 0, 0);
+        assert!(Account[addr].sequence_number == 0, 0);
         increment_sequence_number_for_test(addr); // Increment sequence number
         // Assert correct mock value post-increment
-        assert!(borrow_global<Account>(addr).sequence_number == 1, 1);
+        assert!(Account[addr].sequence_number == 1, 1);
         set_sequence_number(addr, 10); // Set mock sequence number
         // Assert correct mock value post-modification
-        assert!(borrow_global<Account>(addr).sequence_number == 10, 2);
+        assert!(Account[addr].sequence_number == 10, 2);
```

Comment:
could drop the error codes then

Reply:
Dang it didn't drop all the error codes... I didn't check the long tail tests